client/src/pages/Explorer.js

```diff
@@ -73,47 +98,82 @@ function FavoriteCheckBox() {
 
 const ExplorerPage = () => {
   const classes = useStyles();
+  const [generalData, setGeneralData] = useState({places: [], favorites: []});
+  const citiesUrl = "/api/cities/us/?population=1500000";
+  const userFavoriteCitiesUrl = "/api/users/favoriteCities";
+  useEffect(() => {
+    async function getData(){
+      try {
+        const result1 = await fetch(userFavoriteCitiesUrl, {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          },
+        })
+        const result2 = await fetch(citiesUrl, {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          }
+        })
+        const places = await result2.json();
+        const favorites = await result1.json();
+        userFavoritePlaces = Array.isArray(favorites) ? favorites : [];
```

Comment:
What else could `favorites` be? If it's gonna be `null` or `undefined` you can do something like:
`userFavoritePlaces = favourites || [];`

Reply:
Actually it seems like we don't use this variable anywhere? You should be able to remove it

Reply:
> What else could `favorites` be? If it's gonna be `null` or `undefined` you can do something like:
> `userFavoritePlaces = favourites || [];`
We got an Object [something like : {error: "Unaurthorized"}] as response when an unauthenticated user is trying to make the call.  I forgot to mentioned it during the stand up meeting. May be throwing an error is a better approac.

Reply:
> Actually it seems like we don't use this variable anywhere? You should be able to remove it
userFavoritePlaces is used as a global variable to handle user favorite updates. See Line 55 to 72